app/components/Form/ProjectFundingAgreementForm.tsx

```diff
@@ -460,6 +476,30 @@ const ProjectFundingAgreementForm: React.FC<Props> = (props) => {
               Add Funding Source
             </Button>
           </FormBorder>
+          <FormBorder title="Expenses & Payments Tracker">
+            <FormBase
+              id={`expensesPaymentsTracker`}
+              className="expensesPaymentsTrackerForm"
+              idPrefix="expensesPaymentsTracker"
+              schema={expensesPaymentsTrackerSchema as JSONSchema7}
+              formContext={{
+                calculatedEligibleExpensesToDate:
+                  projectRevision.projectFundingAgreementFormChanges?.edges[0]
+                    .node.eligibleExpensesToDate,
+                calculatedHoldbackAmountToDate:
+                  projectRevision.projectFundingAgreementFormChanges?.edges[0]
+                    .node.holdbackAmountToDate,
+                calculatedNetPaymentsToDate:
+                  projectRevision.projectFundingAgreementFormChanges?.edges[0]
+                    .node.netPaymentsToDate,
+                calculatedGrossPaymentsToDate:
+                  projectRevision.projectFundingAgreementFormChanges?.edges[0]
+                    .node.grossPaymentsToDate,
+              }}
+              ObjectFieldTemplate={EmptyObjectFieldTemplate}
+              uiSchema={expensesPaymentsTrackerUiSchema}
+            ></FormBase>
+          </FormBorder>
```

Comment:
` :global(.expensesPaymentsTrackerForm) {
            margin-bottom: 0px;
          }
          :global(.expensesPaymentsTrackerForm > div > div:nth-last-child(2)) {
            margin-bottom: 0px;
          }`
Adding that to the style block at the bottom here helps reduce the extra dead space at the bottom of the new form. I haven't tried it with errors displaying though, so I'll leave it up to you if you want to add it.

Reply:
I'll add it; I don't think we can get errors with a calculated field?

Reply:
No, it wouldn't make sense - unless we want to use the form error pattern to show missing values?